package.json

```diff
@@ -19,16 +19,30 @@
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack --config webpack.conf.js",
-    "serve": "serve dist"
+    "build": "webpack --config ./webpack/webpack.prod.js",
+    "dev": "webpack serve --config ./webpack/webpack.dev.js"
   },
+  "type": "module",
   "devDependencies": {
+    "@babel/core": "^7.17.8",
+    "@babel/preset-env": "^7.16.11",
+    "babel-loader": "^8.2.4",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^10.2.4",
     "css-loader": "^6.5.1",
+    "dat.gui": "^0.7.9",
```

Comment:
Is this really required for a minimal template? Same question for the other packages added here. Are there any others that can be considered uneccessary?

Reply:
dat.gui is not needed for a minimal template, and I will check again. I didn't take care of software quality in production.

Reply:
Do I need to close this PR and open a new one?